### DISCUSSION TITLE
Kodiak 100

### DISCUSSION THREAD

**Discussion by iankbr** [calibx/msfsdeck#65] [General]
If it's any help, the Kodiak 100 has two inside light controls ..
.
1. A THREE-WAY 'Cabin' switch, labelled 'Off'-'Norm'-'On'.
2. An 'Instrument Panel' rotary potentiometer.

The single 'Panel' switch on my Logitech Saitek Switch Panel does absolutely nothing.  I assume that the Kodiak is a little non-standard.

![Kodiak 1000](https://user-images.githubusercontent.com/75181780/155028533-21086daa-8da2-4ecf-b79f-539550c6ef26.png)

Ian.

**Comment by calibx** [comment]
Hi,

If you like me to include something in the plugin, as I don't have the Kodiak 100, you nedd to give me the events ids displayed when you use the trigger.
To do that, go into FSUIPC, and in the log menu, select events. Then just copy here the logs of the events displayed when you trigger the controls

Seb.

**Reply by iankbr** [reply]
I saw on another thread that there was an issue with the cabin light on the Kodiak, so was just trying to help.

Ian.

**Reply by calibx** [reply]
Oh thank you, the problem came from FSUIPC where some cabin lights are inverted and I didn't have identified correctly liveries in the aircraft name.
I think now all is ok since rc7
